Tests/Enquiry/Build/ConfirmSchoolPageTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Application.Common.Models.Enquiry.Build;
+using Application.Validators.Enquiry.Build;
+using FluentValidation.TestHelper;
+using UI.Pages.Enquiry.Build;
+using ConfirmSchool = UI.Pages.Enquiry.Build.ConfirmSchool;
+
+namespace Tests.Enquiry.Build;
+
+[Collection(nameof(SliceFixture))]
+public class ConfirmSchoolPageTests
+{
+    private readonly SliceFixture _fixture;
+
+    public ConfirmSchoolPageTests(SliceFixture fixture) => _fixture = fixture;
+
+    [Fact]
+    public void With_single_true_school_valid()
+    {
+        var model = new ConfirmSchoolModel { HasSingleSchool = true, ConfirmedIsSchool = true };
```

Comment:
The code to create a `ConfirmSchoolModelValidator` and a `ConfirmSchoolModel` is repeated in every test. You could consider moving this to a setup method or the constructor of the test class to reduce duplication.
For example:
```
 private readonly SliceFixture _fixture;
    private readonly ConfirmSchoolModelValidator _validator;

    public ConfirmSchoolPageTests(SliceFixture fixture) 
    {
        _fixture = fixture;
        _validator = new ConfirmSchoolModelValidator();
    }

    private ConfirmSchoolModel CreateModel(bool hasSingleSchool, bool? confirmedIsSchool = null, int? schoolId = null)
    {
        return new ConfirmSchoolModel { HasSingleSchool = hasSingleSchool, ConfirmedIsSchool = confirmedIsSchool, SchoolId = schoolId };
    }

    [Fact]
    public void With_single_true_school_valid()
    {
        var model = CreateModel(true, true);
        var result = _validator.TestValidate(model);
        result.ShouldNotHaveAnyValidationErrors();
    }
```